include/fmt/base.h

```diff
@@ -359,6 +361,10 @@ template <typename... T> FMT_CONSTEXPR void ignore_unused(const T&...) {}
 
 constexpr auto is_constant_evaluated(bool default_value = false) noexcept
     -> bool {
+// If FMT_USE_CONSTEVAL is disabled then this cannot be true
+#if !FMT_USE_CONSTEVAL
+	return false;
+#endif
```

Comment:
Please remove. `is_constant_evaluated` is relevant even if `consteval` is disabled.